docs/release-process.md

```diff
@@ -0,0 +1,46 @@
+# Release Process
+
+The following document covers the release process for the artifacts in this repository.
+
+## Reference Implementation
+
+The following details the release process of the Sidetree reference implementation `@decentralized-identity/sidetree`, which is a typescript based [npm](https://www.npmjs.com/) package.
+
+In general the release process can be summarized by the following flow
+- Each merged pull request into master results in the automated release of a new unstable package on npm, unless the `[skip ci]` tag is included in the commit message.
+- A release commit merged into master triggers a stable release of the reference implementation.
+
+### Stable Releases
+
+To create a stable release follow the following steps
+
+1. Checkout the head of master `git checkout master && git pull`
+2. Create a new release branch from master called `release`
+3. Install the dependencies `npm install`
+5. Build the package `npm build`
+6. Test the package `npm test`
+7. Run `npm run version:release`, with an appropriate option such as [ `major` | `minor` | `patch` ].
```

Comment:
we need a comment here about duplicating a folder / registering the associated typescript modules.

Reply:
@thehenrytsai if you get a chance would be great to contribute more to the vague explanation I have given